### DISCUSSION TITLE
Thin Lattice Girder Tweak

### DISCUSSION THREAD

**Discussion by JimStarluck** [tsunamayo/Starship-EVO#4207] [Suggestions Done]
The way things are currently configured, when placing a block and a thin lattice girder of the same scale, there is a small offset on the lattice girder's thin axis, which means you cannot line it up with the edge of the block:

![image](https://user-images.githubusercontent.com/77904316/134812631-8a09b7bc-1c28-4761-9ddb-45534dbe9857.png)

Personally, I find that far more often, I want to be able to place these two things without that offset, such that the broad side aligns:

![image](https://user-images.githubusercontent.com/77904316/134812685-c923c473-a4fd-42ba-8028-d7cafe7ea1e0.png)

...but in order to do this, I have to drop down a scale for the block, which is not always possible or desirable.

As it stands, the presence of that offset prohibits a lot of the configurations I find myself wanting to use. Among many others, it prevents a seamless connection between the square lattice girder blocks and the thin one:

![image](https://user-images.githubusercontent.com/77904316/134812781-b14d26d5-c439-4d48-8cf7-3f538bfa7f20.png)

Alternatively, I suppose, allowing the thin lattice girder to be offset by up to 1/8th its native grid -- a level finer than the current 1/4th max -- would achieve a similar effect.

**Comment by BigBadKangaroo** [comment]
I pretty much stopped using it because of the tedious offset placement. It really needs an easier way of placeing and lining up properly imo

**Comment by JimStarluck** [comment]
For that matter, it'd be nice if pipes could be routed through them.

**Comment by tsunamayo** [comment]
@JimStarluck @BigBadKangaroo @JimStarluck Suggestion implemented for 21w40a. They also let pipe through. Thanks

**Comment by JimStarluck** [comment]
Awesome! Thank you!
